Gruntfile.js

```diff
@@ -59,6 +59,39 @@ module.exports = function(grunt) {
           spawn: false,
         }
       }
+    },
+    aws: grunt.file.readJSON('./grunt-aws.json'),
+    s3: {
+      options: {
+        key: '<%= aws.key %>',
+        secret: '<%= aws.secret %>',
+        bucket: '<%= aws.bucket %>',
+        access: 'private'
+      },
+      dev: {
+        upload: [
+          {
+            src: 'index.html',
+            dest: 'index.html'
+          },
+          {
+            src: 'login.html',
+            dest: 'login.html'
+          },
+          {
+            src: 'lib/**',
+            dest: 'lib/**'
+          },
+          {
+            src: 'templates/**',
+            dest: 'templates/**'
```

Comment:
The `/templates` folder can be left out since they are compiled and placed in `/public/js/dist`. Same thing for the `/lib` folder.